### PR TITLE
KS: List Item withLeadIcon (Implementation)

### PR DESCRIPTION
**Pull request by alex40724**
Interface was already appreciated and merged.

### CODE REVIEW

File: Services/Component/classes/class.ilPluginSlot.php
Comment (klees) [inline Services/Component/classes/class.ilPluginSlot.php]: Is this really related to the KS-Stuff?

**Review by Amstutz — CHANGES_REQUESTED** [review]
Hi @alex40724 ,

Sorry for the premature merge. Plz get rid of the Plugins stuff here, since there is no obvious relation to the PR and then you can directly merge this after you double checked, that all tests work.

File: Services/Component/classes/class.ilPluginSlot.php
Comment (alex40724) [reply Services/Component/classes/class.ilPluginSlot.php]: No, but this makes the branch installable, see https://www.ilias.de/mantis/view.php?id=23163
I will remove this before merging.

**Pull request merged**
